packages/vector_graphics/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.1.14
+
+* Make the package WASM compatible
```

Comment:
Please see the CHANGELOG style guide linked from the PR checklist; this needs to use the repo-standard verb form and punctuation.

Reply:
updated